metabase/metabase
# handle null value correctly on the row chart

Closes https://github.com/metabase/metabase/issues/16727

## Changes

Fixes handling the `null` value on the row chart.

## How to verify

- Create a structured question with `null` values for the Y-axis of the row chart
- Change its visualization type to the `row` and select the nullable Y-axis and some other column for X-axis
- Ensure the `null` tick is rendered as (empty)
- Hover and ensure the tooltip value shows (empty)
- Drill into the bar and ensure it sets the correct filter `is empty`

<video src="https://user-images.githubusercontent.com/14301985/202034224-818c63cd-de96-48b9-b6ff-74f012c34480.mov" />
